### PR TITLE
Kb/feature/error component

### PR DESCRIPTION
**Pull request by kelsiebesingeryeh**
#### Purpose:
 > Check all that apply.
- [X] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 
Added an error component and route for error handling

Issues closed #

> Describe your code changes in detail for reviewers.
Added an Error component along with styling to display if we have an error loading the data.
Also added a route for the error

Components => App and Error

npm start

### CODE REVIEW

File: src/components/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './Error.sass'
+
+const Error = () => {
+    return (
+        <section className='errorSection'>
```
Comment (gaj23) [inline src/components/Error/Error.js:6]: It'll be good to have this to use! I can see a few places where we might need to apply it.

**Review by gaj23 — APPROVED** [review]
Great! Thank you for this addition.

File: src/components/App/App.js [hunk 2]
```diff
@@ -2,28 +2,19 @@ import React from 'react'
...
-      exact
-      path="/login"
-      render={() => <Login/>} 
-      />
-      <Route 
-      exact
-      path='/user-landing'
-      render={() => <UserLanding/>}
-      />
+      <Route exact path="/" component={Home} />
```
Comment (JeffKersting) [inline src/components/App/App.js:12]: This looks really clean!

**Review by JeffKersting — APPROVED** [review]
This should really help with our end-to-end sad path testing

**Pull request merged**